docs/rest-api/rest-interpreter.md

```diff
@@ -22,28 +22,29 @@ limitations under the License.
 ## Zeppelin REST API
  Zeppelin provides several REST API's for interaction and remote activation of zeppelin functionality.
  
- All REST API are available starting with the following endpoint ```http://[zeppelin-server]:[zeppelin-port]/api```
- 
+ All REST API are available starting with the following endpoint `http://[zeppelin-server]:[zeppelin-port]/api`.
  Note that zeppein REST API receive or return JSON objects, it it recommended you install some JSON view such as 
- [JSONView](https://chrome.google.com/webstore/detail/jsonview/chklaanhfefbnpoihckbnefhakgolnmc)
+ [JSON View](https://chrome.google.com/webstore/detail/jsonview/chklaanhfefbnpoihckbnefhakgolnmc).
  
  
- If you work with zeppelin and find a need for an additional REST API please [file an issue or send us mail](../../community.html) 
+ If you work with zeppelin and find a need for an additional REST API, please [file an issue or send us mail](http://zeppelin.incubator.apache.org/community.html). 
```

Comment:
I would like to do. But currently, there isn't `community.md` in Zeppelin source. So I think we can only access this location using `http://zeppelin.incubator.apache.org/community.html`.